processor/schemaprocessor/processor_metrics_test.go

```diff
@@ -0,0 +1,235 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package schemaprocessor
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+func TestMetrics_RenameAttributes(t *testing.T) {
```

Comment:
From memory, there is a test function that allows you to define this information in a yaml form and compare against an expected yaml file for pdata values.

I suspect this might be more useful here since manually validating each is going to be time consuming.

Reply:
I have updated tests to use yaml. Can you please take a look.